Makefile

```diff
@@ -90,44 +87,61 @@ endif
 CHECK += check-dev
 HELP_check-dev = checks if device is specified
 check-dev:
-	@if [ "$(DEV)" == "none" ]; \
-	then \
-		( \
-		echo "##############################"; \
-		echo "# FLASH DEVICE NOT SPECIFIED #"; \
-		echo "##############################"; \
-		echo;\
-		echo "specify device by adding DEV as parameter";\
-		echo;\
-		echo "make $(MAKECMDGOALS) DEV=/path/to/device";\
-		echo;\
-		echo "or by running";\
-		echo;\
-		echo "make dev";\
-		echo;\
-		) >&2;\
-		exit 1;\
-	fi
-	@if [ ! -c $(DEV) ]; \
-	then \
-		( \
-		echo "#############################"; \
-		echo "# FLASH DEVICE IS NOT VALID #"; \
-		echo "#############################"; \
-		echo; \
-		echo "the specified device ($(DEV)) is not a character device!";\
-		echo;\
-		echo "specify device by adding DEV as parameter";\
-		echo;\
-		echo "make $(MAKECMDGOALS) DEV=/path/to/device";\
-		echo;\
-		echo "or by running";\
-		echo;\
-		echo "make dev";\
-		echo;\
-		) >&2;\
-		exit 1;\
-	fi
+ifeq ($(DEV),none)
+	@1>&2 echo "##############################"
```

Comment:
to hide the echo command you can use ```@echo```, or what's the purpose of @1>&2 here?

Reply:
To output the message at stderr instead of stdout. I was like that before and I wanted to keep it consistent. The idea behind that was probably to be able to log errors? @nopeslide

Reply:
@pascalharp @ikstream 
exactly

Reply:
As there is no logging in the makefile shouldn't we leave output redirection to the user who wants to lots error?

Reply:
@ikstream How should the user log errors if they are not distinguishable from normal output?